docsrc/api.rst

```diff
@@ -11,11 +11,22 @@ Classes
 CmdStanModel
 ============
 
-.. autoclass:: cmdstanpy.CmdStanModel
-   :members:
+A CmdStanModel object encapsulates the Stan program. It manages program compilation and provides the following inference methods:
+
+sample
+    runs the HMC-NUTS sampler to produce a set of draws from the posterior distribution.
 
+optimize
+    produce a penalized maximum likelihood estimate (point estimate) of the model parameters.
 
+variational
```

Comment:
I had a problem running variational inference. I'll open a separate issue for that.